api/src/config.default.json

```diff
@@ -119,7 +119,8 @@
     "can_log_out_on_android": [],
     "can_verify_reports": ["national_admin", "district_admin"],
     "can_view_last_visited_date": [],
-    "can_write_wealth_quintiles": []
+    "can_write_wealth_quintiles": [],
+    "can_view_outgoing_messages": []
```

Comment:
Default this to `national_admin` so it's the same as `can_configure` by default. Same with the new `can_export_everything` one.

Also don't forget to update the standard configuration at `config/standard/app_settings.json`.